data/colors/vars/app_dark.ts

```diff
@@ -3,6 +3,7 @@ import {get, alpha} from '../../../src/utils'
 // Variables to be moved to github/github
 
 export default {
+  canvasDefaultTransparent: alpha(get('canvas.default'), 0),
```

Comment:
I assume that you're adding this in Primitives because github/github doesn't support color functions. Is that correct?

Reply:
Yes, right. There isn't a way to add transparency to `canvas.default`.